src/modules/formBuilder/components/itemEditor/conditionals/CardGroup.tsx

```diff
@@ -21,8 +23,8 @@ export const RemovableCard: React.FC<RemovableCardProps> = ({
         border: '1px solid white',
         borderRadius: 1,
         borderColor: 'borders.light',
-        mb: 1,
```

Comment:
This change does result in a visual change to the `CardGroup`s in the Form Item editor drawer -- they have less spacing between them now. The reason to make this change is so that the margin between the last card and the "Add Condition" button at the bottom is independent of the margin set on the cards themselves.